package.json

```diff
@@ -47,6 +47,7 @@
         "pregenerate-docs": "rimraf docs",
         "generate-docs": "typedoc --options typedoc.json",
         "lint": "eslint --ext .ts src && eslint --plugin mocha test && eslint code_samples && eslint scripts",
+        "lint:fix": "eslint --fix --ext .ts src && eslint --fix --plugin mocha test && eslint --fix code_samples && eslint --fix scripts",
```

Comment:
`npm run lint --fix` === `eslint --ext .ts src && eslint --plugin mocha test && eslint code_samples && eslint scripts --fix`

so it won't fix all the errors

so I added this one.